### PR TITLE
lxd: print the join token on a separate line

### PR DESCRIPTION
**Pull request by simondeziel**
This makes it for easy copy-n-paste using triple click to
select the full text line.

Signed-off-by: Simon Deziel <simon.deziel@canonical.com>

### CODE REVIEW

File: lxc/cluster.go [hunk 1]
```diff
@@ -622,7 +622,8 @@ func (c *cmdClusterAdd) Run(cmd *cobra.Command, args []string) error {
 			return errors.Wrapf(err, "Failed converting token operation to join token")
 		}
 
-		fmt.Printf(i18n.G("Member %s join token: %s")+"\n", resource.name, joinToken.String())
+		fmt.Printf(i18n.G("Member %s join token:")+"\n", resource.name)
+		fmt.Printf(joinToken.String() + "\n")
```
Comment (stgraber) [inline lxc/cluster.go:626]: `fmt.Println(joinToken.String())` would be slightly shorter.

File: lxc/cluster.go [hunk 2]
```diff
@@ -622,7 +622,8 @@ func (c *cmdClusterAdd) Run(cmd *cobra.Command, args []string) error {
 			return errors.Wrapf(err, "Failed converting token operation to join token")
 		}
 
-		fmt.Printf(i18n.G("Member %s join token: %s")+"\n", resource.name, joinToken.String())
+		fmt.Printf(i18n.G("Member %s join token:")+"\n", resource.name)
```
Comment (stgraber) [inline lxc/cluster.go:625]: This changes a translatable string so you need an extra commit:
 - make i18n
 - git commit -m "i18n: Update translation templates" i18n/

^ That should take care of it.

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

**Review by tomponline — APPROVED** [review]

**Comment by stgraber** [comment]
jenkins: test this please

**Pull request merged**